src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/InMemoryExportDestinationClient.cs

```diff
@@ -106,5 +106,10 @@ public string GetExportedData(Uri fileUri)
 
             return null;
         }
+
+        public int GetExportedDataFileCount()
```

Comment:
Looks like it should be a property
`public int ExportedDataFileCount => ...`

Reply:
Changed